creator-node/src/ffprobe-exec.js

```diff
@@ -1,40 +1,35 @@
-const ffprobeStatic = require('ffprobe-static')
-var exec = require('child_process').exec
+const path = require('path')
+const fs = require('fs')
 
 const SEGMENT_REGEXP = /(segment[0-9]*.ts)/
 
 // Retrieve segment durations for an entire directory in a single child process
 // Standard output is parsed and returned as a dictionary of <segmentName, duration>
-async function getSegmentsDuration (req, segmentPath) {
-  let cmd = `find ${segmentPath}/ -maxdepth 1 -iname '*.ts' -print -exec ${ffprobeStatic.path} -v quiet -of csv=p=0 -show_entries format=duration {} \\;`
-
+async function getSegmentsDuration (req, segmentPath, filename, filedir) {
   return new Promise((resolve, reject) => {
```

Comment:
@hareeshnagaraj i don't think this needs to be a promise but i dont think it matters